assets/airports/ktpa.json

```diff
@@ -0,0 +1,1566 @@
+            ["N28d10m14.260", "W82d31m17.899", "N28d11m14.346", "W82d31m15.649"],
+            ["N28d11m14.346", "W82d31m15.649", "N28d11m14.346", "W82d31m15.649"],
+            ["N28d18m14.623", "W82d30m59.890", "N28d19m14.709", "W82d30m57.634"],
+            ["N28d19m14.709", "W82d30m57.634", "N28d19m14.709", "W82d30m57.634"],
+            ["N27d57m31.591", "W82d33m50.889", "N27d58m14.848", "W82d33m04.039"],
+            ["N27d58m14.848", "W82d33m04.039", "N27d58m14.848", "W82d33m04.039"],
+            ["N27d53m24.188", "W82d38m14.681", "N27d53m51.614", "W82d37m42.002"],
+            ["N27d53m51.614", "W82d37m42.002", "N27d53m51.614", "W82d37m42.002"],
+            ["N27d55m24.796", "W82d30m34.229", "N27d56m21.259", "W82d30m57.632"],
+            ["N27d56m21.259", "W82d30m57.632", "N27d56m21.259", "W82d30m57.632"],
+            ["N28d22m26.128", "W82d15m33.372", "N28d21m05.133", "W82d14m41.713"],
+            ["N28d21m05.133", "W82d14m41.713", "N28d21m05.133", "W82d14m41.713"],
+            ["N27d29m59.862", "W83d06m30.073", "N27d10m59.851", "W82d43m59.942"],
+            ["N27d10m59.851", "W82d43m59.942", "N27d14m43.918", "W82d20m02.970"],
+            ["N27d14m43.918", "W82d20m02.970", "N27d14m43.918", "W82d20m02.970"],
+            ["N27d50m14.782", "W82d31m57.529", "N27d51m40.612", "W82d30m35.044"],
+            ["N27d51m40.612", "W82d30m35.044", "N27d51m40.612", "W82d30m35.044"],
+            ["N28d28m49.070", "W82d21m56.829", "N28d28m39.165", "W82d21m56.847"],
+            ["N28d28m39.165", "W82d21m56.847", "N28d28m39.165", "W82d21m56.847"],
+            ["N28d28m39.165", "W82d21m56.847", "N28d28m38.820", "W82d21m45.637"],
+            ["N28d28m38.820", "W82d21m45.637", "N28d28m38.820", "W82d21m45.637"],
+            ["N28d28m36.435", "W82d20m51.093", "N28d28m35.758", "W82d20m39.885"],
+            ["N28d28m35.758", "W82d20m39.885", "N28d28m35.758", "W82d20m39.885"],
+            ["N28d28m23.617", "W82d15m11.129", "N28d28m23.264", "W82d14m59.921"],
+            ["N28d28m23.264", "W82d14m59.921", "N28d28m23.264", "W82d14m59.921"],
+            ["N28d28m20.834", "W82d14m05.382", "N28d28m20.478", "W82d13m54.175"],
+            ["N28d28m20.478", "W82d13m54.175", "N28d28m20.478", "W82d13m54.175"],
+            ["N28d28m07.739", "W82d08m25.818", "N28d28m07.377", "W82d08m14.613"],
+            ["N28d28m07.377", "W82d08m14.613", "N28d28m07.377", "W82d08m14.613"],
+            ["N28d28m05.231", "W82d07m20.075", "N28d28m04.867", "W82d07m08.867"],
+            ["N28d28m04.867", "W82d07m08.867", "N28d28m04.867", "W82d07m08.867"],
+            ["N28d27m51.531", "W82d01m40.542", "N28d27m51.158", "W82d01m29.337"],
+            ["N28d27m51.158", "W82d01m29.337", "N28d27m51.158", "W82d01m29.337"],
+            ["N28d27m48.639", "W82d00m34.806", "N28d27m48.266", "W82d00m23.599"],
+            ["N28d27m48.266", "W82d00m23.599", "N28d27m48.266", "W82d00m23.599"],
+            ["N28d29m47.172", "W82d21m56.734", "N28d29m37.267", "W
```

Comment:
this line has a bunch of spaces at the end, please remove

Reply:
done 👍